### PR TITLE
create comments

### PR DESCRIPTION
**Pull request by Sythrill**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/css
```
Comment (matwjablonski) [inline .gitignore]: if you want to ignore all css files you can just write: 
`*.css`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	<meta charset="utf-8">
+		<title>
+			Tytuł strony
+		</title>
+		<link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+	<h1>
+		Witaj na mojej stronie
+	</h1>
```
Comment (matwjablonski) [inline index.html]: write it in one line, its unnecessary to write html tag with content in three lines when tag hasn't have any attributes

File: js/scripts.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+// plik scripts.js
+//alert("Test")
+
+var myBtn = document.getElementById('info');
+
+
```
Comment (matwjablonski) [inline js/scripts.js]: one empty line is enough :)

File: sass/style.sass [hunk 4]
```diff
@@ -0,0 +1,32 @@
+@mixin rounded($radius)
+	-webkit-border-radius: $radius
+	-moz-border-radius: $radius
```
Comment (matwjablonski) [inline sass/style.sass]: prefixes are unnecessary to border-radius :)

File: sass/style.sass [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+	border-radius: $radius
+
+@mixin gradient($from, $to)
+	background-color: $to 
+	background-image: -moz-linear-gradient($from, $to)
+	background-image: -o-linear-gradient($from, $to)
+	background-image: -webkit-gradient(linear, left top, left bottom, color-stop(0, $from),color-stop(1, $to))
+	background-image: -webkit-linear-gradient($from, $to)
+	background-image: -ms-linear-gradient($from, $to)
+	background-image: linear-gradient($from, $to)
```
Comment (matwjablonski) [inline sass/style.sass]: http://shouldiprefix.com/#gradients

no prefixes :)

File: sass/style.sass [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+%button
+	width: 200px
+	height: 50px
+	padding: 5px
+	color: #fff
+	font-size: 20px
+	border: 0
+
```
Comment (matwjablonski) [inline sass/style.sass]: remove empty line - leave only one empty line between css rules

File: sass/style.sass [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+.button_main
+	@extend %button
+	@include rounded(6px)
+	@include gradient(#42b3e2, #1a6798)
+
+body
+
```
Comment (matwjablonski) [inline sass/style.sass]: like above, empty line to remove

**Comment by Sythrill** [comment]
fixed

**Pull request merged**